ParkJiHan/python/14003.py

```diff
@@ -0,0 +1,40 @@
+# 백준 14003.가장 긴 증가하는 부분 순열 5 (https://www.acmicpc.net/problem/14003)
+
+N = int(input())
+A = list(map(int,input().split()))
```

Comment:
PS에서 많이 사용하는 패턴이고 개인의 취향이 들어가는 영역이긴 합니다만, Python을 프로덕션 환경에서 사용할 계획이 있다면 Functional Programming 방식보다는 List Comprehension에 더 익숙해지는 것이 좋을 것 같습니다
Google Python Style Guide: https://google.github.io/styleguide/pyguide.html#27-comprehensions--generator-expressions
```suggestion
A = [int(x) for x in input().split()]
```

Reply:
python을 프로덕션 레벨에서 사용해본 적이 없어서 궁금했는데, 제시해주신 가이드 등을 공부하면 크게 도움이 될 수 있을 것 같습니다.
적어도 선언부에서라도 comprehension을 더 적극적으로 사용해봐야겠네요. ㅎㅎ
좋은 리뷰 감사합니다. 👍